doc/contrib/mmapstorage.md

```diff
@@ -0,0 +1,18 @@
+# mmapstorage
```

Comment:
Isn't here also information about the cache? Maybe call the file simply `mmap`.

Reply:
IMO the cache is different thing that needs separate docs. The current cache `cache` uses `mmapstorage`, but you can use `mmapstorage` standalone and that has all the same intricacies explained here.